src/notebook/widget.ts

```diff
@@ -337,6 +337,25 @@ class NotebookWidget extends Widget {
 }
 
 
+function createElements(tag: string, attr?: any, children?: HTMLElement[] | string): HTMLElement {
```

Comment:
Also, the name should be `createElement`, since it is only creating the top level element.